build.sh

```diff
@@ -160,8 +160,12 @@ build_managed_corert()
     __buildproj=$__scriptpath/build.proj
     __buildlog=$__scriptpath/msbuild.$__BuildArch.log
 
+    if [ -n ${ToolchainMilestone:+1} ]; then
+        ToolchainMilestone=testing
+    fi
+
     # TODO: Renable running tests
```

Comment:
Can you also take this oppurtunity to remove this TODO and the corresponding "/p:SkipTests=true" so that post-build steps run on Ubuntu and Mac as well?